README.md

```diff
@@ -233,31 +256,66 @@ export default myFunction;
 
 </details>
 
-A function's implementation must be the default export of the source file. 
-When instantiating the function, use `SlackFunctionHandler` and pass 
-in your function's `.definition` so that your required inputs and outputs will be 
-enforced. 
+A function's implementation must be the default export of the source file. When instantiating the function, use `SlackFunctionHandler` and pass in your function's `.definition` so that your required inputs and outputs will be 
+enforced.
 
-## Running your app locally
+## Using the Slack CLI
+To learn more about development with the CLI, you can visit the following guides:
+- [Creating a new app with the CLI](https://api.slack.com/future/create)
+- [Configuring your app](https://api.slack.com/future/manifest)
+- [Developing locally](https://api.slack.com/future/run)
+To view all documentation and guides available for the CLI, visit the [Overview page](https://api.slack.com/future/overview).
 
-While building your app, you can see your changes propagated to your 
-workspace in real-time with `slack run`.
+When using the CLI to develop your app, here are some helpful commands:
+- `slack version`: Checks your current version of the Slack CLI.
+- `slack upgrade`: Checks for updates for current SDK and CLI versions and upgrades them as needed.
+- `slack auth list`: Lists all workspaces the CLI is logged into.
+- `slack auth login`: Allows you to log in to a new or inactive workspace with the CLI.
+To view all other commands available in the CLI, run `slack help`.
 
-Executing `slack run` starts a local development server, syncing changes to 
-your workspace's development version of your app. (You'll know it's the 
-development version because the name has the string `(dev)` appended).
+## Project Structure
 
-Your local development server is ready to go when you see the following:
+### `manifest.ts`
 
-```zsh
-Connected, awaiting events
+`manifest.ts` is a configuration for Slack CLI apps using Deno. This file will establish all basic configurations for your application, including app name and description.
+
+### `slack.json`
+
+`slack.json` is a required file for running Slack CLI apps. This file is a way for the CLI to interact with your project's SDK. It defines script hooks which are *executed by the CLI* and *implemented by the SDK.*
 
+An example of this file's format can be found below:
+```js
+{
+  "hooks": {
+    "get-hooks": "deno run -q --unstable --allow-read --allow-net https://deno.land/x/deno_slack_hooks@0.0.4/mod.ts",
+    // This is a user-defined custom hook that overrides the default "get-manifest"
+    "get-manifest": "deno run -q --unstable --config=deno.jsonc --allow-read --allow-net https://deno.land/x/deno_slack_builder@0.0.8/mod.ts --manifest",
+    // This is a user-defined custom hook that adds new functionality
+    "custom-hook": "deno run my-custom-hook.ts"
+  }
```

Comment:
They have the project that contains an example hook: what value does this snippet serve for the developer that's reading? I'm a little concerned that someone will copy/paste these and be getting outdated versions.